How to Find and Use Technical Resources if You’re New to Tech:proposal.md

```diff
@@ -0,0 +1,60 @@
+
+These weren't too difficult to learn, and if you're not sure where to start, you can start here  to get a better understanding of coding in general. (Sidenote: HTMl is actually a markup language because it doesn't have any programming logic like other languages.)
+
+But if you really want to become more advanced with coding, languages like Javascript and Python could be more up your alley. Javascript makes web designs more interactive, which engages users more than just HTML and CSS. Python is often used for creating web applications, and it is versatile for both small and big projects. 
+
+Depending on what you want to learn, you might better enjoy learning some languages over others. Take a couple minutes to research and gain a better understanding of what skills you hope to build before diving straight into learning. 
+
+**Taking coding lessons with freeCodeCamp**
+ 
+ One great place for you to start as a newbie to coding is [freeCodeCamp](https://www.freecodecamp.org/). This nonprofit organization helps users code through hands-on projects and structured, easy-to-manage lessons. 
+
+In addition to their learning platform, they also have Youtube videos and can connect you to a community of others who are just as excited about coding as you are. 
+
+If you're feeling particularly ambitious, they offer certification programs too in topics like responsive web design and Javascript algorithims and data structures. 
+  
+**Putting your new coding skills to work**
+
+So you've built up some coding skills. Where do you go from here? Now you can begin cool projects that you get to show off later on. These can be as simple as a website portolio that you code from scratch — maybe in a code editor like [CodePen](https://codepen.io/) — or getting a little more complicated like an app that chooses what you eat for dinner. 
+
+Your projects are only limited by your imagination, and even if you're feeling a bit stuck, you can seek out inspiration from others online too. 
+
+**Using W3Schools for coding references** 
+
+Sometimes if you need other examples of coding while you're working on your project, [W3Schools](https://www.w3schools.com/) is a good place to go.They only offer written tutorials, but it's a good reference point. 
+
+Here, you can sort through examples they provide in the coding language you are using to see where your code might be causing problems. You can even open the code editor for each example and try playing around to  gain a better understanding of the concept you're trying to grasp. 
+
+They offer lots of examples in many different languages, so hopefuly it can be helpful to you no matter what you're coding in. 
```

Comment:
I think this sentence can be left out, especially since you already mention the benefit of W3Schools providing lots of examples. Are there any other features of W3Schools you could expand on here?